paderbox/io/audioread.py

```diff
@@ -276,7 +277,7 @@ def recursive_load_audio(
     kwargs = locals().copy()
     path = kwargs.pop('path')
 
-    if isinstance(path, (tuple, list)):
+    if isinstance(path, (tuple, list, types.GeneratorType)):
```

Comment:
Is this related to `get_new_subdir`?

Reply:
No, I committed this some days ago and forgot to push.
Since this is a minor change, I kept it in this PR.